### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by SeuRonao**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `main` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `main` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `main`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://docs.github.com/education/manage-coursework-with-github-classroom/leave-feedback-with-pull-requests)”.
</details>

Subscribed: @ElRiyo

### CODE REVIEW

**Comment by SeuRonao** [comment]
Acabei de ver, no caso teu programa entra em loop e não consegue terminar quando testa com `0.09`.

**Comment by SeuRonao** [comment]
Deadline atingido sem remoção dos erros finais.

File: main.py [hunk 1]
```diff
@@ -4,6 +4,35 @@
...
+            moeda = 0.25
+          elif moeda == 0.25:
+            moeda = 0.10
+          elif moeda == 0.10:
+            moeda = 0.05
+          elif moeda == 0.05:
+            moeda = 0.01
+          totmoe = 0
+          if total == 0:
+            break
```
Comment (SeuRonao) [inline main.py:34]: Você poderia fazer a seguinte mudança nesse trecho de código:
```
    print('Digite o valor do troco, (no maximo R$10.00) :') #entrada de dados
    valor = float(input('> R$ '))
    total = int(valor * 100)
    moeda = 10
    totmoe = 0
    if valor > 1000 or valor == 0:
      print('O valor nao ta disponivel para troco.')
    else:
       while True:
        if total >= moeda:
          total -= moeda
          totmoe += 1
        else:
          if totmoe > 0:
            print(f'Total de {totmoe} moedas de R${moeda/100:.2f}')
          if moeda ==100:
            moeda = 50
          elif moeda == 50:
            moeda = 25
          elif moeda == 25:
            moeda = 10
          elif moeda == 10:
            moeda = 5
          elif moeda == 5:
            moeda = 1
          totmoe = 0
          if total == 0:
            break
```
O problema principal é que está tendo erro por conta da soma de números flutuantes.
No caso, quando você faz as contas para subtrair 0.01 centavo de alguma quantidade, por exemplo, 0.04, o que sobra não é exatamente 0.08 é um pouquinho menos do que isso.
Depois de passar várias vezes no loop para subtrair esse 0.01 o valor  fica 0.009 que não é 0.01 centavo e nem é zero.
Nessa hora ele entra em loop infinito.

Comment (SeuRonao) [reply main.py:34]: O que eu fiz foi fazer as contas de forma que não precise de números flutuantes.

Comment (ElRiyo) [reply main.py:34]: entendi, perfeito. vou modificar isso. obrigado

**Pull request closed without merging**